include/stl2/view/zip_with2.hpp

```diff
@@ -0,0 +1,406 @@
+// cmcstl2 - A concept-enabled C++ standard library
```

Comment:
This new file should be part of a different PR?

Reply:
I have no idea how this was even a part of the commit!